README.md

```diff
@@ -63,11 +63,13 @@ Authenticate to staging API with any of the following credentials
   role: Doctor
 ```
 
-#### 🏷️ Make use labels to update the PR/issue status
+#### Contributing to CARE
 
-- Mark your PRs as `work-in-progress` if it's still being worked on.
-- Once you have solved the related issue, mark your PR with `need testing` and `need review` labels.
-- When you’re making a PR with lots of code changes that affects multiple functionalities, or is likely to break, make sure you tag it with `Major Code Change` label.
+- Create a branch from the latest [`develop`](https://github.com/coronasafe/care_fe/tree/develop) branch when starting to work on an issue.
```

Comment:
Lets add issue number to the branch; 

Examples; 

issues/120/create-beds

Reply:
Okay, but what if there are two closely related issues linked to the same PR?